n_rows_file: 108
n_cols_file: 86
quality_index: 0
file_name: "/db/box_db_2.csv"